Formula/t/tenv.rb

```diff
@@ -21,12 +21,14 @@ class Tenv < Formula
   conflicts_with "opentofu", because: "both install tofu binary"
   conflicts_with "terraform", because: "both install terraform binary"
   conflicts_with "terragrunt", because: "both install terragrunt binary"
+  conflicts_with "atmos", because: "both install atmos binary"
   conflicts_with "tfenv", because: "tfenv symlinks terraform binaries"
   conflicts_with "tgenv", because: "tgenv symlinks terragrunt binaries"
 
   def install
+    ENV["CGO_ENABLED"] = "0"
```

Comment:
```suggestion
```

Reply:
This needs to be removed because it is not relevant to Homebrew builds as @SMillerDev and @chenrui333 have said.

Reply:
Seems fine to set it to make sure our build configuration is closer to what upstream does

Reply:
I don't agree that it's incorrect. We, as authors of tenv want to be sure that tenv on macos will use the same libs as tenv on Alpine.

Reply:
yeah, we should probably remove that line at best. :)

Reply:
why? In this case tenv on macos will work by a different way if we'll compare it with tenv on Alpine